papers/carmina/myst.yml

```
@@ -0,0 +1,42 @@
+version: 1
+extends: ../papers.yml
+project:
+  # Update this to match `scipy-2025-<folder>` the folder should be `<firstname_surname>`
+  id: scipy-2025-carmina
+  # Ensure your title is the same as in your `main.md`
+  title: Carmina
+  subtitle: Introducing Programming to Latin Poetry
+  description: Carmina is a developing Python library designed to help demonstrate the algorithmic nature of poetic analysis processes such as scansion and alliteration analysis. Carmina is written specifically for classicists who may be new to programming, so that they can see the familiar process of scanning Latin reimagined as a computational process.
+  # Authors should have affiliations, emails and ORCIDs if available
+  authors:
+    - name: Suh Young Choi
+      email: atobdura@uw.edu
+      orcid: 0009-0006-2247-2254
+      affiliations:
+        - Department of Classics, University of Washington
+        - Paul G. Allen School of Computer Science and Engineering, University of Washington
+        - Whiting School of Engineering, Johns Hopkins University
+  keywords:
+    - Latin poetry
+    - computational thinking
+    - scansion
+    - alliteration
+  # Add the abbreviations that you use in your paper here
+  abbreviations:
+    MyST: Markedly Structured Text
+  # It is possible to explicitly ignore the `doi-exists` check for certain citation keys
+  error_rules:
+    - rule: doi-exists
+      severity: ignore
+      keys:
+        - Atr03
+        - terradesert
+        - jupyter
+        - sklearn1
+        - sklearn2
```

A few automated checks are failing on citations without DOIs. If these citations actually do not have DOIs, you may add them to this list to prevent the check failure.

```suggestion
        - pattern
        - cltk
        - jupyter
        - sklearn1
        - sklearn2
```